deployment/CP04OSSM_Deploy.csv

```diff
@@ -1,80 +1,80 @@
+AT-27,,AT-31,,CP04OSSM-RIC21-00-CPMENG000,2,1,2015-05-10T01:04:00,2015-10-23T20:52:00,CGMCP-04OSSM-00002,,CGCON-NCPM02-00009,39.94063,-70.87855,,7,442,
+AT-27,,AT-31,,CP04OSSM-RID26-00-DCLENG000,2,1,2015-05-10T01:04:00,2015-10-23T20:52:00,CGMCP-04OSSM-00002,,CGCON-NDCL26-00015,39.94063,-70.87855,,7,442,
+AT-27,,AT-31,,CP04OSSM-RID26-04-VELPTA000,2,1,2015-05-10T01:04:00,2015-10-23T20:52:00,CGMCP-04OSSM-00002,,CGINS-VELPTA-11377,39.94063,-70.87855,,7,442,
+AT-27,,AT-31,,CP04OSSM-RID26-06-PHSEND000,2,1,2015-05-10T01:04:00,2015-10-23T20:52:00,CGMCP-04OSSM-00002,,CGINS-PHSEND-P0155,39.94063,-70.87855,,7,442,
+AT-27,,AT-31,,CP04OSSM-RID26-07-NUTNRB000,2,1,2015-05-10T01:04:00,2015-10-23T20:52:00,CGMCP-04OSSM-00002,,CGINS-NUTNRB-00261,39.94063,-70.87855,,7,442,
+AT-27,,AT-31,,CP04OSSM-RID26-08-SPKIRB000,2,1,2015-05-10T01:04:00,2015-10-23T20:52:00,CGMCP-04OSSM-00002,,CGINS-SPKIRB-00274,39.94063,-70.87855,,7,442,
+AT-27,,AT-31,,CP04OSSM-RID27-00-DCLENG000,2,1,2015-05-10T01:04:00,2015-10-23T20:52:00,CGMCP-04OSSM-00002,,CGCON-NDCL27-00009,39.94063,-70.87855,,7,442,
+AT-27,,AT-31,,CP04OSSM-RID27-01-OPTAAD000,2,1,2015-05-10T01:04:00,2015-10-23T20:52:00,CGMCP-04OSSM-00002,,CGINS-OPTAAD-00240,39.94063,-70.87855,,7,442,
+AT-27,,AT-31,,CP04OSSM-RID27-02-FLORTD000,2,1,2015-05-10T01:04:00,2015-10-23T20:52:00,CGMCP-04OSSM-00002,,CGINS-FLORTD-00994,39.94063,-70.87855,,7,442,
+AT-27,,AT-31,,CP04OSSM-RID27-03-CTDBPC000,2,1,2015-05-10T01:04:00,2015-10-23T20:52:00,CGMCP-04OSSM-00002,,CGINS-CTDBPC-07208,39.94063,-70.87855,,7,442,
+AT-27,,AT-31,,CP04OSSM-RID27-04-DOSTAD000,2,1,2015-05-10T01:04:00,2015-10-23T20:52:00,CGMCP-04OSSM-00002,,CGINS-DOSTAD-00410,39.94063,-70.87855,,7,442,
+AT-27,,AT-31,,CP04OSSM-SBC11-00-CPMENG000,2,1,2015-05-10T01:04:00,2015-10-23T20:52:00,CGMCP-04OSSM-00002,,CGCON-BCPM01-00007,39.94063,-70.87855,,0,442,
+AT-27,,AT-31,,CP04OSSM-SBD11-00-DCLENG000,2,1,2015-05-10T01:04:00,2015-10-23T20:52:00,CGMCP-04OSSM-00002,,CGCON-BDCL11-00007,39.94063,-70.87855,,0,442,
+AT-27,,AT-31,,CP04OSSM-SBD11-01-MOPAK0000,2,1,2015-05-10T01:04:00,2015-10-23T20:52:00,CGMCP-04OSSM-00002,,CGINS-MOPAK0-99999,39.94063,-70.87855,,0,442,
+AT-27,,AT-31,,CP04OSSM-SBD11-02-HYDGN0000,2,1,2015-05-10T01:04:00,2015-10-23T20:52:00,CGMCP-04OSSM-00002,,CGINS-HYDGN0-99999,39.94063,-70.87855,,0,442,
+AT-27,,AT-31,,CP04OSSM-SBD11-06-METBKA000,2,1,2015-05-10T01:04:00,2015-10-23T20:52:00,CGMCP-04OSSM-00002,,CGINS-METLGR-00025,39.94063,-70.87855,,0,442,
+AT-27,,AT-31,,CP04OSSM-SBD12-00-DCLENG000,2,1,2015-05-10T01:04:00,2015-10-23T20:52:00,CGMCP-04OSSM-00002,,CGCON-BDCL12-00007,39.94063,-70.87855,,0,442,
+AT-27,,AT-31,,CP04OSSM-SBD12-03-HYDGN0000,2,1,2015-05-10T01:04:00,2015-10-23T20:52:00,CGMCP-04OSSM-00002,,CGINS-HYDGN0-99999,39.94063,-70.87855,,0,442,
+AT-27,,AT-31,,CP04OSSM-SBD12-04-PCO2AA000,2,1,2015-05-10T01:04:00,2015-10-23T20:52:00,CGMCP-04OSSM-00002,,CGINS-PCO2AA-34229,39.94063,-70.87855,,0,442,
```

Comment:
Reviewed correction to deployment date, recovery time, sensor UIDs and deployment depths. Looks good.